templates/app/views/spree/checkout/steps/_confirm_step.html.erb

```diff
@@ -7,22 +7,18 @@
     <%= render 'spree/orders/order_details', order: @order %>
   </fieldset>
 
-  <% SolidusStarterFrontend::Config[:require_terms_and_conditions_acceptance].tap do |requires_acceptance| %>
-    <% if requires_acceptance %>
-      <div class="terms_and_conditions">
-        <%= render 'spree/checkout/terms_and_conditions' %>
-        <%= check_box_tag :accept_terms_and_conditions, 'accepted', false %>
-        <%= label_tag :accept_terms_and_conditions, t('spree.agree_to_terms_of_service') %>
-      </div>
-    <% end %>
+  <div class="terms_and_conditions">
+    <%= render 'spree/checkout/terms_and_conditions' %>
+    <%= check_box_tag :accept_terms_and_conditions, 'accepted', false %>
+    <%= label_tag :accept_terms_and_conditions, t('spree.agree_to_terms_of_service') %>
+  </div>
 
-    <%= button_tag(
-      I18n.t('spree.place_order'),
-      class: 'button-primary',
-      name: :commit,
-      disabled: requires_acceptance
-    ) %>
-  <% end %>
+  <%= button_tag(
+    I18n.t('spree.place_order'),
+    class: 'button-primary',
+    name: :commit,
+    disabled: true
+  ) %>
```

Comment:
@aldesantis The terms and conditions check box was previously disabled by default. As part of removing the terms and conditions preference, I had two choices:

1. Follow the previous default behavior and delete the check box from the form, or
2. Change the form's behavior and show the check box.

I chose the second option because

1. It's easier for a Solidus developer to delete the check box than to re-add it.
2. I googled "checkout terms and conditions" and it seems that having a terms and conditions checkbox at the checkout is a common requirement. See for example, https://community.shopify.com/c/shopify-design/how-to-add-agreement-checkbox-of-terms-and-conditions-before/td-p/443197 and https://support.thrivecart.com/help/adding-terms-conditions-to-your-checkout-page/.

The downside of taking the second option is that I had to enable JavaScript on some checkout specs in order to enable the Place Order button once the terms and conditions check box is clicked.

Let me know your thoughts!

Reply:
I think enabling it by default is reasonable 👍